routes/web/users.php

```diff
@@ -144,6 +144,14 @@
         ]
     )->name('users.print');
 
+    Route::get(
```

Comment:
This should be a POST request, not a GET. You never want to use a GET request for something that actually *does* something, like changing records, sending emails, etc.